linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_banner_enabled/ansible/shared.yml

```diff
@@ -11,13 +11,18 @@
     value: "true"
     create: yes
     no_extra_spaces: yes
+  register: result_ini
```

Comment:
I have tried to run a per-rule playbook for this rule against a RHEL 9 VM and I confirm that it's idempotent successfully.